src/three/TilesRenderer.js

```diff
@@ -675,6 +676,29 @@ export class TilesRenderer extends TilesRendererBase {
 			cached.scene = scene;
 			cached.metadata = metadata;
 
+			if ( ! this.loading && this.parseQueue.items.length > 0 ) {
+
+				if ( this.onLoadStart ) {
+
+					this.onLoadStart( );
+
+				}
+				this.loading = true;
+
+			}
```

Comment:
It occurs to me that this will not trigger if we're in a case where only one tile has downloaded - because this condition will be checked only once a file has been fully parsed. Perhaps this "onStart" condition should go at the end of the update function, instead?